pkgs/data/fonts/noto-fonts/default.nix

```
@@ -110,25 +111,29 @@ in
   };
 
   noto-fonts-emoji = let
-    version = "unstable-2019-10-22";
+    version = "unstable-2020-08-20";
+    emojiPythonEnv =
+      python3.withPackages (p: with p; [ fonttools nototools ]);
   in stdenv.mkDerivation {
     pname = "noto-fonts-emoji";
     inherit version;
 
     src = fetchFromGitHub {
-      owner = "googlei18n";
+      owner = "googlefonts";
       repo = "noto-emoji";
-      rev = "018aa149d622a4fea11f01c61a7207079da301bc";
-      sha256 = "0qmnnjpp5lza6g5m3ki6hj46p891h9vl42k3acd0qw8i0jj5yn2c";
+      rev = "1bc491419fa2925d018f27bfe702792031be0e68";
+      sha256 = "1vak4s1p4wlwzpnqfb1c2sg62q82gnjpnmqrfz8xl6bd0z55imzy";
     };
 
-    buildInputs = [ cairo ];
-    nativeBuildInputs = [ pngquant optipng which cairo pkgconfig imagemagick ]
-                     ++ (with python3Packages; [ python fonttools nototools ]);
+    nativeBuildInputs = [ cairo imagemagick zopfli pngquant which
+                          pkgconfig emojiPythonEnv ];
```

most common formatting with multiline lists
```suggestion
    nativeBuildInputs = [
      cairo
      imagemagick
      zopfli
      pngquant
      which
      pkgconfig
      emojiPythonEnv
    ];
```

---

Done in 84dfeee7d823bc2e0744b703681ae891526cd0e6.